libs/hdf-converters/src/gosec-mapper.ts

```diff
@@ -47,7 +47,7 @@ function formatMessage(input: Record<string, unknown>): string {
   return `${_.get(input, 'confidence')} confidence of rule violation at:\n${_.get(input, 'code')}`;
 }
 
-export class GoSecMapper extends BaseConverter {
+export class GosecMapper extends BaseConverter {
```

Comment:
This might not be a change we can do without breaking a downstream project's stuff.

Reply:
I've reached out to @jweissm to see if they can provide any input.

Reply:
They didn't respond.  We move forward.